packages/components/src/VirtualizedList/CellTextIcon/CellWithIcon.component.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Action from '../../Actions/Action';
+import styles from './CellWithIcon.scss';
+
+/**
+ * Cell renderer that displays a badge
+ */
+function CellWithIcon(props) {
+	const action = props.columnData.getIcon(props.rowData);
+
+	return (
+		<div className={styles['cell-icon-container']}>
```

Comment:
please use something like `classNames('cell-icon-container', styles['cell-icon-container'])` so style can be customized elsewhere.